pootle/templates/admin/dashboard.html

```diff
@@ -57,30 +57,19 @@
 
 <div id="depchecks" class="module" lang="{{ LANGUAGE_CODE }}">
   <div class="hd">
-    <h2>{% trans "Dependency Checks" %}</h2>
+    <h2>{% trans "System Checks" %}</h2>
   </div>
   <div class="bd">
-    <ul id="required" class="icons">
-      {% for dep in required %}
-      <li class="{{ dep.dependency }}"><i class="icon-{{ dep.state }}"></i> {{ dep.text }}</li>
-      {% endfor %}
+    <ul>
+    {% for check in checks %}
+      <li>
+        <i class="icon-{% if check.is_serious %}reject{% else %}warning{% endif %}"></i>
+        {{ check.msg }} <em>{{ check.hint }}</em>
+      </li>
+    {% empty %}
+      <li><i class="icon-accept"></i> {% trans "No issue detected." %}</li>
```

Comment:
Let's use `icon-tick` as before.

Also, *issues*.
Anyhow, being more verbose can reassure the system checks passed: *No issues have been detected. All is good!*